src/Tests/dotnet-workload-install.Tests/MockPackWorkloadInstaller.cs

```diff
@@ -159,20 +165,24 @@ public void ReplaceWorkloadResolver(IWorkloadResolver workloadResolver)
             WorkloadResolver = workloadResolver;
         }
 
-        public void DeleteInstallState(SdkFeatureBand sdkFeatureBand)
+        public void RemoveManifestsFromInstallState(SdkFeatureBand sdkFeatureBand)
         {
             string path = Path.Combine(WorkloadInstallType.GetInstallStateFolder(sdkFeatureBand, _dotnetDir), "default.json");
             if (File.Exists(path))
             {
-                File.Delete(path);
+                var installStateContents = InstallStateContents.FromString(File.Exists(path) ? File.ReadAllText(path) : "{}");
```

Comment:
More instances where we can do `new InstallStateContents()` instead of parsing `{}`.